CHANGELOG.rst

```diff
@@ -7,6 +7,193 @@ Changelog
+
+    We improved the logs that are emitted by the application, which enable better integration with observability
+    tooling like Grafana.
+
+    TODO-5398: should we mention that we also prototyped a dashboard? Can people even access this?
+
+**🔒️ Authentication using Yivi and eIDAS**
+
+    We implemented support for `Yivi <https://yivi.app/>`_ and `eIDAS <https://en.wikipedia.org/wiki/EIDAS>`_
+    authentication through OIDC. With support for Yivi authentication, end-users can decide which personal
+    information they want to share with Open Forms.
+
+    Support for eIDAS will allow European citizens without a DigiD (and/or BSN) to have access to forms which
+    require authentication.
+
+**👫 Partners component with prefill**
+
+    We added a partners component, where information such as initials, last name, and date of birth of a
+    partner can be shown.
+
+    This component can be prefilled using a new family members prefill plugin, that allows retrieving data from
+    "Haal Centraal BRP personen bevragen" (version 2) or "StUF-BG" (version 3.1).
+
+**📝 JSON schema generation**
+
+    We added the possibility to generate a JSON schema of a form. It describes the submission data of all user-defined
+    and component variables, and can be generated for the Generic JSON and Objects API registrations as if the data
+    were sent through either of those plugins.
+
+    The schemas of component variables also include a description and validation rules if they were specified
+    in the configuration of these components.
+
+Detailed changes
+----------------
+
+**New features**
+
+* [:backend:`4966`, :backend:`5285`, :backend:`5334`] Improved the logs emitted by the application to better integrate
+  with observability tooling like Grafana.
+
+* [:backend:`5140`] Reworked the authentication module architecture to make it possible to add support for
+  new plugins based on the OpenID Connect protocol (Yivi and eIDAS).
+
+* [:backend:`5132`] Added support for authentication using Yivi through OIDC.
+
+    - Allows logging in to forms using DigiD, eHerkenning, or anonymously.
+    - Additional attribute groups can be defined in the Yivi configuration, and relevant ones can be selected per form.
+      These groups allow end-users to, optionally, provide additional personal or company details.
+
+# TODO-5398: add bullet points
+* [:backend:`4453`] Added support for authentication using eIDAS through OIDC.
+
+    - Amazing feature 1
+    - Amazing feature 2
+
+* [:backend:`5254`] Added new family members prefill plugin.
+
+    - The data can be retrieved from "Haal Centraal BRP personen bevragen" (version 2) or "StUF-BG" (version 3.1).
+    - Partners or children of the authenticated user can be stored in a user-defined variable.
+    - The retrieved data of children can be filtered by age and whether they are deceased.
+
+* [:backend:`4944`, :backend:`5268`, :sdk:`824`, open-formulieren/formio-builder#225] Added partners component.
```

Comment:
```suggestion
* [:backend:`4944`, :backend:`5268`, :sdk:`824`] Added partners component.
```
builder/renderer tickets are invisible components and don't need to be linked in the changelog. Backend + SDK are our "public interfaces/components/APIs".